### DISCUSSION TITLE
Kedro 0.18.0

### DISCUSSION THREAD

**Discussion by bkolasa** [kedro-org/kedro#1117] [Idea]
Hello!

I'm looking forward to use Kedro in my organisation, but unfortunately it is crucial for us to support Python 3.9. I read that this support will be introduced in Kedro 0.18.0. Do you know what is an estimate time of release of that version for now?

**Comment by antonymilne** [comment]
We're hoping (but no guarantee, sorry!) to bring out 0.18 in Q1 next year, so hopefully there shouldn't be long to wait. However, **unofficially** I actually think that in most cases it should be straightforward for you to use kedro 0.17.6 already with Python 3.9:
1.  Create your Python 3.9 environment
2. `pip install kedro --ignore-requires-python` to install kedro 0.17.6 while ignoring the Python version constraint
3. If you have any datasets that are `pandas.ExcelDataSet`: `pip install openpyxl` and add `engine: openpyxl` to `load_args` in the catalog entry

I just tried this out with the [spaceflights starter](https://github.com/quantumblacklabs/kedro-starters/tree/main/spaceflights) and all worked fine. The only bits in `src/requirements.txt` that weren't compatible with Python 3.9 were the `kedro*` ones. I just commented out those lines, did `pip install -r src/requirements.txt` and then installed the `kedro*` requirements using `--ignore-requires-python`.

**Comment by datajoely** [comment]
Hey @bkolasa check out 0.18.0 when you get a chance!
